src/components/Layout/HomeContent.js

```diff
@@ -131,100 +131,100 @@ export function HomeContent() {
           <Center>
-            <Header>Create user interfaces from components</Header>
+            <Header>Crie interfaces de usuário a partir de componentes</Header>
             <Para>
-              React lets you build user interfaces out of individual pieces
-              called components. Create your own React components like{' '}
-              <Code>Thumbnail</Code>, <Code>LikeButton</Code>, and{' '}
-              <Code>Video</Code>. Then combine them into entire screens, pages,
-              and apps.
+              O React permite que você crie interfaces de usuário a partir de
+              peças individuais chamadas de componentes. Crie seus próprios
+              componentes React como <Code>Thumbnail</Code>,{' '}
+              <Code>LikeButton</Code>, e <Code>Video</Code>. Em seguida,
+              combine-os em telas inteiras, páginas, e aplicativos.
             </Para>
           </Center>
           <FullBleed>
             <Example1 />
           </FullBleed>
           <Center>
             <Para>
-              Whether you work on your own or with thousands of other
-              developers, using React feels the same. It is designed to let you
-              seamlessly combine components written by independent people,
-              teams, and organizations.
+              Quer você trabalhe sozinho ou com milhares de outros
+              desenvolvedores, usar o React parece o mesmo. Ele foi projetado
+              para permitir que você combine componentes escritos por pessoas,
+              equipes e organizações independentes.
             </Para>
           </Center>
         </Section>
 
         <Section background="right-card">
           <Center>
-            <Header>Write components with code and markup</Header>
+            <Header>Escrever componentes com código e marcação</Header>
             <Para>
-              React components are JavaScript functions. Want to show some
-              content conditionally? Use an <Code>if</Code> statement.
-              Displaying a list? Try array <Code>map()</Code>. Learning React is
-              learning programming.
+              Componentes React são funções JavaScript. Quer mostrar alguns
+              conteúdos condicionalmente? Utilize <Code>if</Code>. Exibindo uma
+              lista? Tente um array <Code>map()</Code>. Aprender React é
+              aprender programação.
             </Para>
           </Center>
           <FullBleed>
             <Example2 />
           </FullBleed>
           <Center>
             <Para>
-              This markup syntax is called JSX. It is a JavaScript syntax
-              extension popularized by React. Putting JSX markup close to
-              related rendering logic makes React components easy to create,
-              maintain, and delete.
+              Essa sintaxe de marcação é chamada de JSX. É uma extensão de
+              sintaxe JavaScript popularizada pelo React. Colocar a marcação JSX
```

Comment:
```suggestion
              sintaxe JavaScript popularizada pelo React. Colocar tags JSX
```

O termo `markup` não tem uma tradução em português. No PR #660 traduzi ele como `tag` e na maioria dos casos pareceu bom.
Minha sugestão é usar ele para manter uma uniformidade.